src/mobile/src/main/java/tupperdate/android/home/RecipeCard.kt

```diff
@@ -0,0 +1,63 @@
+package tupperdate.android.home
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.Text
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.imageResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import tupperdate.android.R
+import tupperdate.android.ui.TupperdateTypography
+import tupperdate.api.RecipeApi
+
+@Composable
+fun recipeCard(offset: Int, recipe: RecipeApi.Recipe, boxWidth: Dp) {
+    Card(
+        modifier = Modifier.fillMaxSize().offset(x = offset.dp),
+        backgroundColor = (Color.Cyan),
+        shape = RoundedCornerShape(8.dp)
+    ) {
+        Box(Modifier.fillMaxSize())
+        {
+            Image(
+                // TODO modify parameter to use a String url
```

Comment:
Not needed for this PR, but you may want to use the https://github.com/chrisbanes/accompanist library to display images in compose.